src/renderers/dom/client/wrappers/ReactDOMSelect.js

```diff
@@ -58,6 +58,28 @@ function warnIfValueIsNull(props) {
   }
 }
 
+function warnIfDuplicateValues(inst) {
+  const options = inst._currentElement.props.children;
```

Comment:
They can also be nested inside arrays/maps/sets etc, so this does not work reliably.